pnnl.goss.gridappsd/scripts/fncs_goss_bridge.py

```diff
@@ -209,18 +277,27 @@ def _registerWithGOSS(username,password,gossServer='localhost',
         raise ValueError(
             'stompPort must be a nonempty string.\n' 
             + 'stompPort = {0}'.format(stompPort))
+    global gossConnection
     gossConnection = stomp.Connection12([(gossServer, stompPort)])
     gossConnection.start()
-    gossConnection.connect(username,password)
+    gossConnection.connect(username,password, wait=True)
     gossConnection.set_listener('GOSSListener', GOSSListener())
     gossConnection.subscribe(input_from_goss_topic,1)
+    gossConnection.subscribe(input_from_goss_queue,2)
+
+    logger.info('registered with goss on topic '+input_from_goss_topic+' '+str(gossConnection.is_connected()))
+
+def _keepAlive():
+    while 1:
+        time.sleep(0.1) 
     
 if __name__ == "__main__":
     #TODO: send simulationId, fncsBrokerLocation, gossLocation, 
     #stompPort, username and password as commmand line arguments 
-    _registerWithGOSS('system','password',gossServer='localhost',stompPort='61613')
-    _registerWithFncsBroker('simulation1','tcp://localhost:5570')
-    
-    
-    
+    simulationId = sys.argv[1]
```

Comment:
Consider using argparse for parsing arguments to the script rather than hard coding values.